### PR TITLE
Typofix202403

### PR DESCRIPTION
**Pull request by cpinotossi**

### CODE REVIEW

File: 03-Azure/01-03-Infrastructure/08_Migration_Datacenter_Linux/README.md [hunk 1]
```diff
@@ -177,7 +177,7 @@ The goal of this exercise is to create a business case. The Business case capabi
 
 - On-premises vs Azure total cost of ownership.
 - Year on year cashflow analysis.
-- Resource utilization based insights to identify servers and workloads that are ideal for cloud.
+- Resource utilization based insights to identdeify servers and workloads that are ideal for cloud.
```
Comment (ajendreieckm) [inline 03-Azure/01-03-Infrastructure/08_Migration_Datacenter_Linux/README.md:180]: It was correct. Identify and not Identdeify.

File: 03-Azure/01-03-Infrastructure/08_Migration_Datacenter_Linux/walkthrough/challenge-1/solution.md [hunk 2]
```diff
@@ -15,27 +15,41 @@ flowchart LR
...
+    task3["Task#3
+    Verify the Azure deployment"]
+    task4["Task#4
+    Verify Web Server availability"]
+    task1 --> task2
+    task2 --> task3
+    task3 --> task4
+~~~
+
 Duration: 30 minutes
```
Comment (ajendreieckm) [inline 03-Azure/01-03-Infrastructure/08_Migration_Datacenter_Linux/walkthrough/challenge-1/solution.md:33]: It's duplicated. Please remove one.

File: 03-Azure/01-03-Infrastructure/08_Migration_Datacenter_Linux/walkthrough/challenge-1/solution.md [hunk 3]
```diff
@@ -15,27 +15,41 @@ flowchart LR
...
 
 ## Prerequisites
 
+- Please ensure that you successfully verified the [General prerequisits](../../README.md#general-prerequisites) before continuing with this challenge.
 - Please ensure that you successfully verified the [General prerequisits](../../README.md#general-prerequisites) before continuing with this challenge.
```
Comment (ajendreieckm) [inline 03-Azure/01-03-Infrastructure/08_Migration_Datacenter_Linux/walkthrough/challenge-1/solution.md:38]: Also duplicated. Please remove one.

File: 03-Azure/01-03-Infrastructure/08_Migration_Datacenter_Linux/walkthrough/challenge-1/solution.md [hunk 4]
```diff
@@ -69,6 +83,9 @@ az login
 az login --use-device-code # If you are using a browser on a different machine
 ~~~
 
+![Login from remote shell with device code](./img/mh.linux.login.gif)
+
+
 ![Login from remote shell with device code](./img/mh.linux.login.gif)
```
Comment (ajendreieckm) [inline 03-Azure/01-03-Infrastructure/08_Migration_Datacenter_Linux/walkthrough/challenge-1/solution.md:89]: also duplicated. Please remove one.

File: 03-Azure/01-03-Infrastructure/08_Migration_Datacenter_Linux/walkthrough/challenge-1/solution.md [hunk 5]
```diff
@@ -80,30 +97,32 @@ az account list
...
-prefix="51" # replace sm with your own prefix
+prefix="1Team" # replace sm with your own prefix
 suffix=a # replace a with your own suffix
 # Have a look at all possible azure location with Azure CLI
 az account list-locations -o table # List all possible Azure locations
 # Define location for all azure resources
 location=germanywestcentral
 # Get our Entra ID Object ID, will be needed to assign us Admin rights to the azure VMs
 currentUserObjectId=$(az ad signed-in-user show --query id -o tsv)
+currentUserObjectId=$(az ad signed-in-user show --query id -o tsv)
```
Comment (ajendreieckm) [inline 03-Azure/01-03-Infrastructure/08_Migration_Datacenter_Linux/walkthrough/challenge-1/solution.md:116]: duplicated.

File: 03-Azure/01-03-Infrastructure/08_Migration_Datacenter_Linux/walkthrough/challenge-2/solution.md [hunk 6]
```diff
@@ -15,6 +15,21 @@ flowchart LR
...
+    Deploy Azure Windows VM"]
+    task3["Task#3
+    Install Azure Migration Appliance"]
+    task4["Task#4
+    Configure Azure Migration Appliance"]
+    task1 --> task2
+    task2 --> task3
+    task3 --> task4
+~~~
+
```
Comment (ajendreieckm) [inline 03-Azure/01-03-Infrastructure/08_Migration_Datacenter_Linux/walkthrough/challenge-2/solution.md:32]: duplicated.

File: 03-Azure/01-03-Infrastructure/08_Migration_Datacenter_Linux/walkthrough/challenge-2/solution.md [hunk 7]
```diff
@@ -77,6 +96,7 @@ Wait until the deployment has been successfully completed and select *Go to reso
 
 ![image](./img/AzMigApp5.png)
 
+#### [OPTIONAL] Create the Azure Migrate Appliance VM via Azure Resource Manager Template (Bicep) instead of the Azure Portal
 #### [OPTIONAL] Create the Azure Migrate Appliance VM via Azure Resource Manager Template (Bicep) instead of the Azure Portal
```
Comment (ajendreieckm) [inline 03-Azure/01-03-Infrastructure/08_Migration_Datacenter_Linux/walkthrough/challenge-2/solution.md:100]: duplicated

File: 03-Azure/01-03-Infrastructure/08_Migration_Datacenter_Linux/walkthrough/challenge-2/solution.md [hunk 8]
```diff
@@ -112,6 +132,7 @@ userObjectId=$currentUserObjectId \
 imageReference="$migVmImageReference"
 ~~~
 
+#### Log into the Azure Migrate Appliance VM via Azure Bastion
```
Comment (ajendreieckm) [inline 03-Azure/01-03-Infrastructure/08_Migration_Datacenter_Linux/walkthrough/challenge-2/solution.md:135]: duplicated

File: 03-Azure/01-03-Infrastructure/08_Migration_Datacenter_Linux/walkthrough/challenge-2/solution.md [hunk 9]
```diff
@@ -123,6 +144,9 @@ Select *Bastion* from the navigation pane on the left, provide the credentials t
 
 ### **Task 3: Install the Azure Migrate Appliance**
 
+> [!IMPORTANT] You will need to install Microsof Edge Browser or Chrome on the Azure Migrate Appliance VM. Internet Explorer 11 is not supported.
```
Comment (ajendreieckm) [inline 03-Azure/01-03-Infrastructure/08_Migration_Datacenter_Linux/walkthrough/challenge-2/solution.md:147]: broken schema somewhere. It does not look like the purple important block.

**Review by ajendreieckm — CHANGES_REQUESTED** [review]

File: 03-Azure/01-03-Infrastructure/08_Migration_Datacenter_Linux/walkthrough/challenge-1/solution.md (hunk 2 above)
Comment (cpinotossi) [reply 03-Azure/01-03-Infrastructure/08_Migration_Datacenter_Linux/walkthrough/challenge-1/solution.md]: done

File: 03-Azure/01-03-Infrastructure/08_Migration_Datacenter_Linux/walkthrough/challenge-1/solution.md (hunk 3 above)
Comment (cpinotossi) [reply 03-Azure/01-03-Infrastructure/08_Migration_Datacenter_Linux/walkthrough/challenge-1/solution.md]: done

File: 03-Azure/01-03-Infrastructure/08_Migration_Datacenter_Linux/walkthrough/challenge-1/solution.md (hunk 4 above)
Comment (cpinotossi) [reply 03-Azure/01-03-Infrastructure/08_Migration_Datacenter_Linux/walkthrough/challenge-1/solution.md]: done

File: 03-Azure/01-03-Infrastructure/08_Migration_Datacenter_Linux/walkthrough/challenge-1/solution.md (hunk 5 above)
Comment (cpinotossi) [reply 03-Azure/01-03-Infrastructure/08_Migration_Datacenter_Linux/walkthrough/challenge-1/solution.md]: done

File: 03-Azure/01-03-Infrastructure/08_Migration_Datacenter_Linux/walkthrough/challenge-2/solution.md (hunk 6 above)
Comment (cpinotossi) [reply 03-Azure/01-03-Infrastructure/08_Migration_Datacenter_Linux/walkthrough/challenge-2/solution.md]: done

File: 03-Azure/01-03-Infrastructure/08_Migration_Datacenter_Linux/walkthrough/challenge-2/solution.md (hunk 7 above)
Comment (cpinotossi) [reply 03-Azure/01-03-Infrastructure/08_Migration_Datacenter_Linux/walkthrough/challenge-2/solution.md]: done

File: 03-Azure/01-03-Infrastructure/08_Migration_Datacenter_Linux/walkthrough/challenge-2/solution.md (hunk 9 above)
Comment (cpinotossi) [reply 03-Azure/01-03-Infrastructure/08_Migration_Datacenter_Linux/walkthrough/challenge-2/solution.md]: done

**Review by ajendreieckm — APPROVED** [review]

**Pull request merged**